packages/compass-import-export/src/export/export-utils.spec.ts

```diff
@@ -0,0 +1,156 @@
+/* eslint-disable mocha/max-top-level-suites */
```

Comment:
Is this a useful lint to have? I wonder if we should just disable it if it's not helping us prevent anything.

Reply:
We probably should remove it. That just leads to an extra level of nesting for no good reason.

Reply:
(but I think we can file a ticket and do that as a follow-up)